sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob/ci.yml

```diff
@@ -0,0 +1,64 @@
+trigger:
+  branches:
+    include:
+      - main
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob/
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+  paths:
+    include:
+      - sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob/
+
+stages:
```

Comment:
1. Why is this so different than any other pipeline?
2. Why not in your service directory as is normal for a family of crates? They get tested together typically.

Reply:
It's not actualy used, I believe.